recipes/sdl_image/all/conanfile.py

```diff
@@ -47,23 +52,35 @@ class SDLImageConan(ConanFile):
         "pnm": True,
         "svg": True,
         "tga": True,
+        "qoi": True,
         "xcf": True,
         "xpm": True,
         "xv": True,
         "with_libjpeg": True,
         "with_libtiff": True,
         "with_libpng": True,
         "with_libwebp": True,
+        "with_avif": False,
+        "with_jxl": False,
         "imageio": False,
+        "wic": False,
     }
 
-    exports_sources = "CMakeLists.txt"
+    def export_sources(self):
+        if Version(self.version) < "2.6":
+            copy(self, "CMakeLists.txt", src=self.recipe_folder, dst=self.export_sources_folder)
 
     def config_options(self):
         if self.settings.os == "Windows":
             del self.options.fPIC
         if not is_apple_os(self):
             del self.options.imageio
+        if Version(self.version) < "2.6":
+            if self.settings.os != "Windows":
+                del self.options.wic
```

Comment:
Just to confirm, wic has been an option all this time but we were just missing it right?